static/src/javascripts/projects/common/modules/audio/index.js

```diff
@@ -48,9 +48,7 @@ const init = (): void => {
         const downloadUrl = placeholder.dataset.downloadUrl;
         const duration = placeholder.dataset.duration;
 
-        if (supportsCSSGrid) {
-            sendToOphan(mediaId, 'ready');
```

Comment:
will all the other events be fired on browsers that do not support the player?

Reply:
yes